Makefile

```diff
@@ -127,15 +127,27 @@ build: airflow-build
 
 build-dev: airflow-dev-build
 
+clean:
+	$(DOCKER_COMPOSE) down -v
+
+airflow-initdb:
+	$(DOCKER_COMPOSE) run --rm  webserver initdb
+
+
+end2end-test:
+	$(MAKE) clean
+	$(MAKE) airflow-initdb
```

Comment:
do we still need `airflow-initdb` here if it is in `run_test.sh`?

Reply:
removed